### DISCUSSION TITLE
Amibian / Amiberry support

### DISCUSSION THREAD

**Discussion by lukas-remis** [keirf/greaseweazle#116] [General]
Hi,
Would it work with raspberry Pi emulators, like Amibian or Amiberry ?

Cheers,
Lukas

**Comment by keirf** [comment]
The UAE emulation support? Only if ported.

**Reply by lukas-remis** [reply]
That would be a great solution.

**Comment by ninjaweb62** [comment]
It looks like both Amiberry and WinUAE added greaseweazle support.

**Reply by keirf** [reply]
This is great news :)
